src/types/multi-select.ts

```diff
@@ -1,3 +1,4 @@
+import { type PopoverAttributes } from './popover'
 export interface MultiselectItem {
```

Comment:
Let's make value a generic type:

```ts
export interface MultiselectItem<T extends string = string> {
  value: T
}
```

This way, the types of `modelValue` and `items` can be more accurate and stay in sync.